app/models/concerns/orchestration/compute.rb

```diff
@@ -74,6 +74,7 @@ def setCompute
     logger.info "Adding Compute instance for #{name}"
     add_interfaces_to_compute_attrs
     self.vm = compute_resource.create_vm compute_attributes.merge(:name => vm_name, :provision_method => provision_method)
+    return true
```

Comment:
Why do we need this? If create_vm is successful, then self.vm would be returned whose truthiness should be a valid return, no?